docs/commands.md

```diff
@@ -211,7 +211,12 @@ TS.MRANGE fromTimestamp toTimestamp [AGGREGATION aggregationType bucketSizeSecon
 
 * fromTimestamp - Start timestamp for range query
 * toTimestamp - End timestamp for range query
-* filters - Set of key-pair fitlers (`k=v`, `k!=v,` `k=` contains a key, `k!=` doesn't contain a key)
+* filters - Set of key-pair fitlers. The supported filters are:
+  * `k=v` label equals value
+  * `k!=v` label doesn't equal value
+  * `k=` key contains the label `k`
```

Comment:
@danni-m if I'm not mistaken, this is the opposite
'k=' means key doesn't contain label and
'k!=' means key does contain label, no?

Reply:
Not confusing at all....

Reply:
Anyway, I didn't change the content just the formatting (i.e. new lines), so we can open a separate bug about this, and decide what the correct behaviour is.